addons/account/views/account_view.xml

```diff
@@ -752,7 +752,7 @@
             <field name="model">account.bank.statement.line</field>
             <field name="priority">8</field>
             <field name="arch" type="xml">
-                <tree string="Statement lines" create="false">
+                <tree string="Statement lines" create="false" default_order="date desc, statement_id desc, id desc">
```

Comment:
shouldn't be `sequence` used somewhere in `default_order`?

Reply:
Hmmm... I had removed it from the original to put statement_id instead, but we actually can keep it after statement_id, so that the lines keep the order of their originating statement, indeed.